prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'JunLee';
+    console.log(fullName);
+    var numberOfPets = 1;
+    console.log(numberOfPets);
+    var likesVegetables = true;
+    console.log(likesVegetables);
+    var nothing = 'null';
+    console.log(nothing);
```

Comment:
Hi Jun, for numbers, booleans, and null, you can leave out the `''` quotes. When JavaScript sees quotes, it makes the value a string.